package.json

```diff
@@ -31,7 +31,7 @@
     "hoek": "3.x.x"
   },
   "peerDependencies": {
-    "react": "0.14.x ",
+    "react": "15.0.x",
     "react-dom": "0.14.x "
```

Comment:
I think the build is failing because `react-dom` in `peerDependencies` should also be `15.x.x`